packages/runtime/container-runtime/src/test/gc/gcConfigs.spec.ts

```diff
@@ -942,7 +867,7 @@ describe("Garbage Collection configurations", () => {
 			configProvider.set(throwOnTombstoneLoadOverrideKey, true);
 			gc = createGcWithPrivateMembers(
 				undefined /* metadata */,
-				{ enableGCSweep: true, [gcDisableThrowOnTombstoneLoadOptionName]: true },
```

Comment:
In line with my comment about the "override" logic (which isn't very override-y in this PR), let's add a test case where the "override" is set to true but `enableGCSweep` is `false`, to show that it can't actually turn on Throw if Sweep is disabled.  It's only useful for disabling the Throw part of Tombstone phase.

Or change the config logic :)

Reply:
Changed the config as per your other comment.